core/src/main/java/com/sequenceiq/cloudbreak/service/ServiceEndpointCollector.java

```diff
@@ -49,6 +50,9 @@
 
     private static final String API_TOPOLOGY_POSTFIX = "-api";
 
+    @Value("${cb.https.port}")
+    private String httpsPort;
```

Comment:
Do we need this?

Reply:
I think yes, nginx will be listening on that port for public facing.